packages/manager/modules/pci/src/projects/project/sidebar/sidebar.html

```diff
@@ -64,8 +66,10 @@ <h2 class="mb-0 d-inline-block" style="font-weight: 400;" data-ng-bind="$ctrl.pr
                 <span class="sr-only" data-translate="cloud_sidebar_public_cloud_project_toggle_close"></span>
             </button>
         </div>
-        <div class="w-100 text-truncate ml-3">
-            <h2 class="mb-0 d-inline-block text-white" style="font-weight: 400;" data-ng-bind="$ctrl.project.description"></h2>
+        <div class="w-100 ml-3 minw-0">
+            <div class="w-100 text-truncate">
+                <h2 class="mb-0 d-inline-block text-white" style="font-weight: 400;" data-ng-bind="$ctrl.project.description"></h2>
```

Comment:
Inline style could be also moved to `_sidebar.less` I guess.

Reply:
@antleblanc I used bootstrap utils instead